apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -83,6 +91,19 @@ export default class ResourceList extends Component {
     );
   };
 
+  sendLinkVisitedEvent = (resource, visitType) => {
+    analyticsReporter.sendEvent(EVENTS.LESSON_RESOURCE_LINK_VISITED_EVENT, {
+      resourceId: resource.id,
+      resourceName: resource.name,
+      resourceAudience: resource.audience,
+      resourceType: resource.type,
+      resourceUrl: resource.url,
+      resourceDownloadUrl: resource.download_url,
+      visitType: visitType,
+      path: document.location.pathname
```

Comment:
@dju90 I'm just making up all these keys, since I wasn't sure what would be useful here. We don't have access to the lesson data in this component, unfortunately. Is there anything in here you don't want, or anything else I should try to add in?

Reply:
I can add the referrer for this page, but it will be blank if they clicked through from the course overview, and it'd be a duplicate from the page view event since these are on that same page. Did you want the referrer for the external links (which would be the current, lesson plan page)? That's what `path` represents here, but I can put the full URL instead if that's better.

Reply:
Took a closer look, I think path is fine actually! Lgtm